### PR TITLE
IR HiFi amplifier control & switch on/off

### PR DESCRIPTION
**Pull request by deltazerorsan**
## Description:
Control via IR a servo that moves the knob. Also decode one IR key to send a mqtto message to Node-RED and to a smart plug via http

**Related issue (if applicable):** fixes <link to issue>

**Pull request in [esphome](https://github.com/esphome/esphome) with YAML changes (if applicable):** esphome/esphome#<esphome PR number goes here>

## Checklist:

  - [x] Branch: `next` is for changes and new documentation that will go public with the next ESPHome release. Fixes, changes and adjustments for the current release should be created against `current`.
  - [x] Link added in `/index.rst` when creating new documents for new components or cookbook.

### CODE REVIEW

File: cookbook/ESPHome-IR-amplifier-control.rst [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+      board: esp-wrover-kit
+    
+      on_boot:
+        priority: -10
+        then:
+          - servo.write:
+              id: my_servo
+              level: 0                   # put the servo in neutral (stoped)
+          - delay: 100ms
+          - servo.detach: my_servo       # always detach the servo to ensure it stops completely
```
Comment (glmnet) [inline cookbook/ESPHome-IR-amplifier-control.rst:72]: this is a bug currently in v1.14, should work ok in next version (dev) you might want to review this when updating

**Comment by glmnet** [comment]
This is missing the link in /index.rst so others can reach the cookbook.

You can preview your work here: https://deploy-preview-619--esphome.netlify.app/cookbook/esphome-ir-amplifier-control

**Comment by deltazerorsan** [comment]
> This is missing the link in /index.rst so others can reach the cookbook.
> 
> You can preview your work here: https://deploy-preview-619--esphome.netlify.app/cookbook/esphome-ir-amplifier-control

Yes you are right, also the image for the index was missed.

Now is checked, thanks.

File: cookbook/ESPHome-IR-amplifier-control.rst (hunk 1 above)
Comment (deltazerorsan) [reply cookbook/ESPHome-IR-amplifier-control.rst:72]: Ok, I will try when it is solved.

**Pull request closed without merging**